### PR TITLE
Obstacle avoidance planner 0.8

### PR DESCRIPTION
**Pull request by k0suke-murakami**

### CODE REVIEW

**Comment by wep21** [comment]
@k0suke-murakami 
https://github.com/k0suke-murakami/Pilot.Auto/blob/799d173270f693acd5cd2453f53cb31413f7512f/planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/src/node.cpp#L187-L191
Are vehicle parameters supposed to be loaded from vehicle info utils?

**Comment by k0suke-murakami** [comment]
@wep21 Thanks I updated the way to get `vehicle_info`

> @k0suke-murakami
> https://github.com/k0suke-murakami/Pilot.Auto/blob/799d173270f693acd5cd2453f53cb31413f7512f/planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/src/node.cpp#L187-L191
> Are vehicle parameters supposed to be loaded from vehicle info utils?

**Comment by wep21** [comment]
@k0suke-murakami 
Could you add vehicle info param path into launch file?

File: planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/src/node.cpp [hunk 1]
```diff
@@ -133,6 +134,14 @@ ObstacleAvoidancePlanner::ObstacleAvoidancePlanner()
     declare_parameter("acceleration_for_non_deceleration_range", 1.0);
   traj_param_->max_dist_for_extending_end_point =
     declare_parameter("max_dist_for_extending_end_point", 5.0);
+  traj_param_->is_avoiding_unknown = declare_parameter("avoiding_object_type/unknown", true);
```
Comment (wep21) [inline planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/src/node.cpp:137]: A delimiter of nested parameter should be ".", so "avoiding_object_type/unknown" should be "avoiding_object_type.unknown".

Comment (k0suke-murakami) [reply planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/src/node.cpp]: Thanks! I pushed the commit to modify the nested param.

**Comment by wep21** [comment]
@k0suke-murakami `colcon test --packages-select obstacle_avoidance_planner` failed. Coulyd you check lint?

**Review by wep21 — APPROVED** [review]
LGTM

**Pull request merged**